project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 0.13.15
+sbt.version = 0.13.16
```

Comment:
Regardless, I can drop it from here, it's not necessary and can be a separate PR.